### DISCUSSION TITLE
Any tips for calibrating the Z-Axis on setting the step/mm

### DISCUSSION THREAD

**Discussion by frank7790** [inkstitch/inkstitch#1463] [Q&A]
Hi Everyone,
I am currently trying to put a CNC embroider however I'm having a little trouble with getting the correct step/mm for the $102 value in GRBL. Should I be treating it like a drill that moves up? 

The minimum value to move the needle is 3mm @ $102=133.330 step/mm
Z-axis motor micro step is set at 1/32 or 6400 pulse/rev

Any help would be appreciated?

Thank you

**Comment by Blackbumpkin** [comment]
Hi,
unfortunately I'm not quite there yet, I've only got the electrics in place.  did you already get it?  If yes, I would be very interested in the result.

**Comment by mgn-norm** [comment]
Idk if either of you ever figured it out, but what I had to do was take and set $102 to 1/4 of the step of 6400 to 1600 and then when exporting in inkstitch set the z travel to 4 for a full revolution. I'm not sure if that's the correct way but it's what is working for me.
